calm_adapter/terraform/deletion_checker_worker.tf

```diff
@@ -0,0 +1,58 @@
+module "calm_deletions_checker_queue" {
+  source                     = "git::github.com/wellcomecollection/terraform-aws-sqs//queue?ref=v1.1.2"
+  queue_name                 = "calm-deletions-checker"
+  topic_arns                 = [local.calm_deletion_checker_topic_arn]
+  aws_region                 = local.aws_region
+  alarm_topic_arn            = local.dlq_alarm_arn
+  visibility_timeout_seconds = 10 * 60 * 60
+}
+
+module "deletion_checker_worker" {
+  source = "../../infrastructure/modules/worker"
+
+  name = local.namespace
+
+  image = local.calm_adapter_image
+
+  env_vars = {
+    calm_api_url             = local.calm_api_url
+    deletion_checker_sqs_url = module.calm_deletions_checker_queue.url
+    adapter_sns_topic        = module.calm_adapter_topic.arn
```

Comment:
I would use a separate topic for the adapter and deletion worker. It doesn't cost any extra, and being able to tap the outputs separately can be useful.

Reply:
Good point (that will require another subscription in the transformer though)